src/devcycle.js

```diff
@@ -0,0 +1,55 @@
+import { initializeDevCycle } from "@devcycle/js-client-sdk";
+
+const user = {
+    // Edit the user properties here to see how they interact with your feature targeting rules
+    user_id: "user1",
+    email: "firstuser@email.com",
+};
+
+const newUser = {
+    // Try editing your feature's targeting rules in the Development environment to serve Variation Off to this user 
+    // then click the "Identify New User" button to see the new user's experience
+    // Note: targeting rules are evaluated from top to bottom, so the first rule that matches will be used
+    user_id: "user2",
+    email: "newuser@email.com",
+};
+
+// Initialize the DevCycle client with your SDK key and user
+const devcycleOptions = { logLevel: "debug" };
+const devcycleClient = initializeDevCycle(
+    process.env.DEVCYCLE_CLIENT_SDK_KEY,
+    user,
+    devcycleOptions
+);
+
+
+devcycleClient.onClientInitialized().then(({ config }) => {
+    console.log("DevCycle client initialized with configuration: ", config);
```

Comment:
do we actually want this log? the sdk seems to be fairly verbose with the logging on init

Reply:
true, i'll drop it